examples/multi-runner/templates/runner-configs/windows-x64.yaml

```
@@ -13,6 +12,7 @@ runner_config:
     - m5.large
     - c5.large
   runners_maximum_count: 1
+  delay_webhook_event: 0
```

can you set the delay to 5, this example is non ephemeral. This allows current workers to start the job when idle.
```suggestion
  delay_webhook_event: 5
```